core/storage/buffer_map.hpp

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_BUFFER_MAP_HPP
+#define KAGOME_BUFFER_MAP_HPP
+
+/**
+ * This file contains:
+ *  - BufferMap - contains key-value bindings of Buffers
+ *  - PersistedBufferMap - stores key-value bindings on filesystem or remote
+ * connection.
+ */
+
+#include <gsl/span>
+
+#include "common/buffer.hpp"
+#include "storage/concept/generic_map.hpp"
+#include "storage/concept/persisted_map.hpp"
+#include "storage/concept/write_batch.hpp"
+
+namespace kagome::storage {
+
+  using Buffer = kagome::common::Buffer;
```

Comment:
Do not declare such using in hpp

Reply:
But I intentionally put it here, so consumers can change type of Buffer any time, and below there will be no long types